build-a-brain/supplementary.rst

```diff
@@ -0,0 +1,39 @@
+***********************
+Supplementary materials
+***********************
+
+Chapter 1
+=========
+
+.. image:: feynmann-blackboard.jpg
+   :target: http://archives-dc.library.caltech.edu/islandora/object/ct1%3A551
+
+Richard Feynman's Blackboard at the time of his death.
+Provided by `Caltech Archives <http://archives.caltech.edu/>`_.
+
+Chapter 2
+=========
+
+Post synaptic time constants for various neurotransmitters
+with supporting citations can be found on `this page
+<http://compneuro.uwaterloo.ca/research/constants-constraints/neurotransmitter-time-constants-pscs.html>`_.
+
+Chapter 3
+=========
+
+The 6 DOF physical arm model can be `downloaded here
+<http://compneuro.uwaterloo.ca/research/motor-control/6-muscle-3-link-arm-model.html>`_.
```

Comment:
Hmm... right now the only places that I can find this are on the research pages and the model archive -- it's not a Nengo model though, so I'm not sure if that'll get moved along with it... will talk to @studywolf about where to put it more permanently, so I just added a todo comment for now.